### DISCUSSION TITLE
Any update on progress towards transformer support?

### DISCUSSION THREAD

**Discussion by JBartholomewMN** [TheStageAI/TorchIntegral#29] [Q&A]
I know that we previously discussed this in an issue I raised, but I'm wondering if you can provide an update on how the integration and support for transformer-type models is coming?

Thanks.

**Comment by JBartholomewMN** [comment]
bump

**Reply by JBartholomewMN** [reply]
bump

**Reply by JBartholomewMN** [reply]
bump

**Comment by b1n0** [comment]
Thanks for awaiting, it is in progress, it needs 2-3 weeks more.
